model/Export/AbstractQTIItemExporter.php

```diff
@@ -140,7 +143,14 @@ public function export($options = [])
                         $stream = $mediaSource->getFileStream($link);
                     }
 
-                    $baseName = ($mediaSource instanceof LocalItemSource) ? $link : 'assets/' . $mediaSource->getBaseName($link);
+                    $baseName = $link;
+
+                    if ($mediaSource instanceof MediaSource) {
+                        $baseName = $this->copySharedStimulusCssFiles($link, $basePath);
+                    } elseif (!$mediaSource instanceof LocalItemSource) {
```

Comment:
What will happen when media source is nether `LocalItemSource` nor `MediaSource`?